server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -269,17 +267,25 @@ export function configureWebSocketServices(
 		async function connectDocument(message: IConnect): Promise<IConnectedClient> {
 			const startTime = Date.now();
 			const throttleErrorPerCluster = checkThrottleAndUsage(
-				connectThrottlerPerCluster,
+				throttlersMap?.get("generalCluster")?.get("socketConnections"),
 				getSocketConnectThrottleId("connectDoc"),
 				message.tenantId,
 				logger,
 			);
 			if (throttleErrorPerCluster) {
 				return Promise.reject(throttleErrorPerCluster);
 			}
+
+			const tenantGroup: string | undefined = message.tenantId
+				? tenantThrottlersMap?.get(message.tenantId) ?? undefined
+				: undefined;
 			const throttleErrorPerTenant = checkThrottleAndUsage(
-				connectThrottlerPerTenant,
-				getSocketConnectThrottleId(message.tenantId),
+				tenantGroup
```

Comment:
Can we change this
`const throttleErrorPerTenant = checkThrottleAndUsage(
				tenantGroup
					? throttlersMap?.get(tenantGroup)?.get("socketConnections")
					: throttlersMap?.get("generalTenant")?.get("socketConnections"),
				tenantGroup
					? `${getSocketConnectThrottleId(message.tenantId)}_${tenantGroup}`
					: getSocketConnectThrottleId(message.tenantId),
				message.tenantId,
				logger,
			);`

to
`const connectThrottlerPerTenant = tenantGroup
					? throttlersMap?.get(tenantGroup)?.get("socketConnections")
					: throttlersMap?.get("generalTenant")?.get("socketConnections");

const connectThrottlerPerTenantId = tenantGroup
					? `${getSocketConnectThrottleId(message.tenantId)}_${tenantGroup}`
					: getSocketConnectThrottleId(message.tenantId);

const throttleErrorPerTenant = checkThrottleAndUsage(
				connectThrottlerPerTenant,
				connectThrottlerPerTenantId,
				message.tenantId,
				logger,
			);`

basically moving the null checks(?:) out of the checkThrottleAndUsage(). This will make the code more readable.

Reply:
Looks like github didn't format all the code. let me know if my comment seems confusing.

Reply:
Done. :)